packages/front-end/components/Features/CodeSnippetModal.tsx

```diff
@@ -34,11 +34,12 @@ export function getApiBaseUrl(connection?: SDKConnectionInterface): string {
     );
   }
 
+  //TODO: We should be able to remove this if we add an env variable to our cloud instance
   if (isCloud()) {
     return `https://cdn.growthbook.io`;
   }
 
-  return trimTrailingSlash(getApiHost());
+  return trimTrailingSlash(getCdnHost()) || trimTrailingSlash(getApiHost());
```

Comment:
`getCdnHost()` may return null and that will break `trimTrailingSlash`.  Probably better to do:

```suggestion
  return trimTrailingSlash(getCdnHost() || getApiHost());
```